website/Makefile

```diff
@@ -62,6 +65,20 @@ $(latest_scaladoc_dir)/index.html: $(latest_version) | $(latest_scaladoc_dir)
 	  rm -rf temp.jar
 	touch $@
 
+$(latest_chisel_download):
+	mkdir -p $@
+
+$(latest_chisel_download)/chisel-example.scala: $(latest_chisel_download) $(latest_version)
```

Comment:
`latest_chisel_download` is an [order-only prerequisite](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html) (only existence matters, not timestamp):

``` suggestion
$(latest_chisel_download)/chisel-example.scala: $(latest_version) | $(latest_chisel_download) 
```